rules/windows/defense_evasion_execution_msbuild_started_by_script.toml

```diff
@@ -61,3 +63,10 @@ id = "TA0002"
 name = "Execution"
 reference = "https://attack.mitre.org/tactics/TA0002/"
 
+[rule.new_terms]
+field = "new_terms_fields"
+value = ["user.name", "process.command_line"]
```

Comment:
Do you think this could cause any FPs with local accounts such as Administrators?

Reply:
Maybe with could add host.name or agent.id into the mix